src/components/interfaces/HMI_API.xml

```diff
@@ -4766,6 +4776,11 @@
     <param name="appID" type="Integer" mandatory="true">
       <description>ID of application that concerns this RPC.</description>
     </param>
+    <param name="cancelID" type="Integer" mandatory="false">
```

Comment:
Realizing that this may need to be added to VR.PerformInteraction as well.

Reply:
It was in VR.PerformInteraction but missing from UI.ScrollableMessage. This was fixed in https://github.com/smartdevicelink/sdl_core/pull/2963/commits/6774f3475722d88f6c552823c07ed36b8ac8ea62